extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -51,12 +61,14 @@ export class LocalSubscriptionPlatform {
    * @return {!Promise<!Entitlements>}
    */
   getEntitlements() {
-    user().assert(this.serviceConfig_['paywallUrl'],
-        'Service config does not have paywall Url');
     const currentProductId = user().assertString(
         this.pageConfig_.getProductId(), 'Current Product ID is null');
+
     return this.xhr_
-        .fetchJson(this.serviceConfig_['paywallUrl'])
+        .fetchJson(this.authorizationUrl_,{
+          mode: 'cors',
```

Comment:
No need for `mode` here. Found this: https://github.com/ampproject/amphtml/blob/master/extensions/amp-access/0.1/amp-access-client.js#L126

Reply:
Done